app/javascript/packs/one-time-code-input.js

```diff
@@ -1,6 +1,12 @@
 import OneTimeCodeInput from '@18f/identity-one-time-code-input';
 
-const input = document.querySelector('.one-time-code-input');
-if (input) {
-  new OneTimeCodeInput(/** @type {HTMLInputElement} */ (input)).bind();
+const fakeField = document.querySelector('.one-time-code-input');
+const input = document.querySelector('.hidden-input');
```

Comment:
Similar to what's done below for the `OneTimeCodeInput` constructor argument, you can move the type-cast to the assignment, which would fix the current lint failure where TypeScript doesn't know what type of element is returned by `querySelector`:

```suggestion
const fakeField = /** @type {HTMLInputElement?} */ (document.querySelector('.one-time-code-input'));
const input = /** @type {HTMLInputElement?} */ (document.querySelector('.hidden-input'));
```

Then you can simplify the constructor below:

```js
new OneTimeCodeInput(fakeField).bind();
```